### PR TITLE
Update change log to include breaking changes for release

### PR DESCRIPTION
**Pull request by cormacpayne**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@cormacpayne__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (corm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: ChangeLog.md [hunk 1]
```diff
@@ -1,10 +1,150 @@
 ##2016.08.03 version 2.0.0
+* Removal of Force parameters
```
Comment (markcowl) [inline ChangeLog.md]: Removed Force parameters that were obsoleted in the previous release

File: ChangeLog.md [hunk 2]
```diff
@@ -1,10 +1,150 @@
...
+
+  * StreamAnalytics
+    - Remove-AzureRmStreamAnalyticsFunction
+    - Remove-AzureRmStreamAnalyticsInput
+    - Remove-AzureRmStreamAnalyticsJob
+    - Remove-AzureRmStreamAnalyticsOutput
+
+  * Tag
+    - Remove-AzureRmTag
+* Change of Tag parameters
```
Comment (markcowl) [inline ChangeLog.md]: Changed 'Tags' parameter name to 'Tag' and changed parameter type from Hashtable[] to HashTable for these cmdlets

File: ChangeLog.md [hunk 3]
```diff
@@ -1,10 +1,150 @@
...
+  * TrafficManager
+    - New-AzureRmTrafficManagerProfile
 * Azure Redis Cache
   * New cmdlet added for New-AzureRmRedisCacheScheduleEntry 
   * New cmdlet added for New-AzureRmRedisCachePatchSchedule 
   * New cmdlet added for Get-AzureRmRedisCachePatchSchedule
   * New cmdlet added for Remove-AzureRmRedisCachePatchSchedule
 * Azure Storage
+  * Get-AzureRmStorageAccountKey
+    - Cmdlet now returns a list of keys, rather than an object whose properties can access each key
```
Comment (markcowl) [inline ChangeLog.md]: rather than an object with properties for each key

File: ChangeLog.md [hunk 4]
```diff
@@ -1,10 +1,150 @@
...
+  * New-AzureRmStorageAccountKey
+    - `StorageAccountRegenerateKeyResponse` field in output of this cmdlet is renamed to `StorageAccountListKeysResults`, which is now a list of keys rather than an object whose properties can access each key
```
Comment (markcowl) [inline ChangeLog.md]: rather than an object with properties for each key

File: ChangeLog.md (hunk 1 above)
Comment (markcowl) [inline ChangeLog.md]: As First bullet:  This release contains breaking change.  Please see the migration guide here (link to the gist) for details.

Comment (markcowl) [inline ChangeLog.md]: As first bullet: this release contains breaking changes see [the migration guide]: link to your document for details on the changes and how existing scripts are impacted.

File: ChangeLog.md [hunk 5]
```diff
@@ -1,10 +1,129 @@
 ##2016.08.03 version 2.0.0
+* Removal of Force parameters that were obsoleted in the previous release
```
Comment (markcowl) [inline ChangeLog.md]: First bullet:

This release contains breaking changes.  Please see [the migration guide](link to your gist) for change details and the impact on existing scripts.

Comment (markcowl) [inline ChangeLog.md]: ... that were marked as obsolete in the ...

File: ChangeLog.md [hunk 6]
```diff
@@ -1,10 +1,129 @@
...
+    - Remove-AzureStorageShareStoredAccessPolicy
+    - Remove-AzureStorageTableStoredAccessPolicy
+  * StreamAnalytics
+    - Remove-AzureRmStreamAnalyticsFunction
+    - Remove-AzureRmStreamAnalyticsInput
+    - Remove-AzureRmStreamAnalyticsJob
+    - Remove-AzureRmStreamAnalyticsOutput
+  * Tag
+    - Remove-AzureRmTag
+* Change of `Tags` parameter name to `Tag`, and change of the parameter type from `HashTable[]` to `HashTable` for the following cmdlets
```
Comment (markcowl) [inline ChangeLog.md]: Changed 'Tags' parameter ... changed parameter type

**Pull request merged**